qutebrowser/config/config.py

```diff
@@ -209,6 +209,8 @@ class ConfigManager(QObject):
         ('colors', 'tab.indicator.stop'): 'tabs.indicator.stop',
         ('colors', 'tab.indicator.error'): 'tabs.indicator.error',
         ('colors', 'tab.indicator.system'): 'tabs.indicator.system',
+        ('tabs', 'always-hide'): 'hide-always',
```

Comment:
Okay, last one before I'll test and (if it works for me) merge ;)

I'd say let's remove this one. Except for you, there's not going to be anyone who had the old name in the config for a just invented option ;)